lib/create_or_edit_brew.dart

```diff
@@ -8,6 +8,64 @@ import 'extensions.dart';
 import 'constants.dart';
 
 import 'package:coffee_journal/model/brew.dart';
+                    RawAutocomplete<Brew>(
```

Comment:
Please fix this. This should be the third child in the Column widget in the `_buildForm` method.

Reply:
Remove this please.